src/main/scala/org/biodatageeks/sequila/apps/SequilaApp.scala

```diff
@@ -0,0 +1,20 @@
+package org.biodatageeks.sequila.apps
+
+import org.apache.spark.sql.{SequilaSession, SparkSession}
+import org.biodatageeks.sequila.utils.SequilaRegister
+
+trait SequilaApp {
+  def createSequilaSession(): SequilaSession = {
+    System.setProperty("spark.kryo.registrator", "org.biodatageeks.sequila.pileup.serializers.CustomKryoRegistrator")
+    val spark = SparkSession
+      .builder()
+      .master("local[1]")
+      .config("spark.driver.memory","4g")
```

Comment:
maybe move it to input args with some low-default value like 1-2g

Reply:
issue? :)